spec/python-spec.coffee

```diff
@@ -245,3 +245,35 @@ describe "Python grammar", ->
     expect(tokens[0][1].scopes).toEqual ['source.python']
     expect(tokens[0][2].value).toBe 'foo'
     expect(tokens[0][2].scopes).toEqual ['source.python']
+
+  it "tokenizes comments inside function parameters", ->
+    {tokens} = grammar.tokenizeLine('def test(arg, # comment')
+    expect(tokens.length).toBe 10
```

Comment:
I don't think we need this (only the above two tests use it and I don't think any of the other languages do).